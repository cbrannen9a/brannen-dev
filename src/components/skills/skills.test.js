import React from 'react';
import renderer from 'react-test-renderer';

import Skills from './skills';

const skills = [{
	id: 'skill1', title: 'isSkill1', _rawDescription: [{
		style: 'normal', _key: 'block1', _type: 'block', markDefs: [],
		children: [
			{
				marks: [],
				text: 'test text',
				_key: 'mock1',
				_type: 'span'
			},
			{
				marks: ['strong'],
				text: 'test bold',
				_key: 'mock2',
				_type: 'span'
			},
			{
				marks: ['em'],
				text: 'test italic',
				_key: 'mock3',
				_type: 'span'
			}]
	}]
},
{
	id: 'skill2', title: 'isSkill2', _rawDescription: [{
		style: 'normal', _key: 'block1', _type: 'block', markDefs: [],
		children: [
			{
				marks: [],
				text: 'test text',
				_key: 'mock1',
				_type: 'span'
			},
			{
				marks: ['strong'],
				text: 'test bold',
				_key: 'mock2',
				_type: 'span'
			},
			{
				marks: ['em'],
				text: 'test italic',
				_key: 'mock3',
				_type: 'span'
			}]
	}]
}];

describe('Skills', () => {
	it('renders correctly', () => {
		const tree = renderer
			.create(<Skills skills={skills} />)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});