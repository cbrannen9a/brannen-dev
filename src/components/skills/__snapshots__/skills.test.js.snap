// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skills renders correctly 1`] = `
<span
  className="tags"
  style={
    Object {
      "marginTop": "5px",
    }
  }
>
  <div
    style={
      Object {
        "marginRight": "4px",
      }
    }
  >
    <span
      className="tooltip"
      onMouseLeave={[Function]}
    >
      <span
        className="tooltip-trigger"
        onMouseOver={[Function]}
      >
        <span
          className="tag"
        >
          isSkill1
        </span>
      </span>
    </span>
  </div>
  <div
    style={
      Object {
        "marginRight": "4px",
      }
    }
  >
    <span
      className="tooltip"
      onMouseLeave={[Function]}
    >
      <span
        className="tooltip-trigger"
        onMouseOver={[Function]}
      >
        <span
          className="tag"
        >
          isSkill2
        </span>
      </span>
    </span>
  </div>
</span>
`;
